torchvision/csrc/models/alexnet.h

```diff
@@ -0,0 +1,24 @@
+#ifndef ALEXNET_H
+#define ALEXNET_H
+
+#include <torch/torch.h>
+
+namespace vision {
+namespace models {
+// AlexNet model architecture from the
+// "One weird trick..." <https://arxiv.org/abs/1404.5997> paper.
+class AlexNetImpl : public torch::nn::Module {
```

Comment:
I think it would be fine to change all impls to `struct` instead of `class` and omit the `public` specifier. People will eventually want to access these members that are currently private.

Reply:
Yes it would. I'll change this